conf/test_host_rm.config

```diff
@@ -0,0 +1,27 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run nf-core/mag -profile test_host_rm_fa,<docker/singularity>
+ */
+
+params {
+  config_profile_name = 'Test profile'
+  config_profile_description = 'Minimal test dataset to check pipeline function'
+  // Limit resources so that this can run on GitHub Actions
+  max_cpus = 2
+  max_memory = 6.GB
+  max_time = 48.h
+  host_fasta = "https://github.com/nf-core/test-datasets/raw/mag/host_reference/genome.hg38.chr21_10000bp_region.fa"
+  readPaths = [
+    ['test_minigut_hg38host', ['https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_hg38host_R1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_hg38host_R2.fastq.gz']],
+    ['test_minigut_sample2_hg38host', ['https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_hg38host_R1.fastq.gz', 'https://github.com/nf-core/test-datasets/raw/mag/test_data/test_minigut_sample2_hg38host_R2.fastq.gz']]
+  ]
+  centrifuge_db = "https://github.com/nf-core/test-datasets/raw/mag/test_data/minigut_cf.tar.gz"
+  kraken2_db = "https://github.com/nf-core/test-datasets/raw/mag/test_data/minigut_kraken.tgz"
+  skip_krona = true
```

Comment:
Any particular reason for skipping this in this test?

Reply:
No, just copied it from `test.config`. If it should not be skipped I guess it would make more sense to turn it on in the main `test.config`, or?

Reply:
Than probably I had a particular reason not to do this but do not remember!
It might be not bad to produce a krona plot in one of the tests, but because I might be the source of the problem here, this shouldn't be your problem.